src/routes/organizations/organization-safes.controller.ts

```diff
@@ -0,0 +1,125 @@
+import { RowSchema } from '@/datasources/db/v2/entities/row.entity';
+import { AuthPayload } from '@/domain/auth/entities/auth-payload.entity';
+import { Organization } from '@/domain/organizations/entities/organization.entity';
+import { Auth } from '@/routes/auth/decorators/auth.decorator';
+import { AuthGuard } from '@/routes/auth/guards/auth.guard';
+import {
+  CreateOrganizationSafeDto,
+  CreateOrganizationSafesSchema,
+} from '@/routes/organizations/entities/create-organization-safe.dto.entity';
+import {
+  DeleteOrganizationSafeDto,
+  DeleteOrganizationSafesSchema,
+} from '@/routes/organizations/entities/delete-organization-safe.dto.entity';
+import { GetOrganizationSafeResponse } from '@/routes/organizations/entities/get-organization-safe.dto.entity';
+import { OrganizationSafesService } from '@/routes/organizations/organization-safes.service';
+import { ValidationPipe } from '@/validation/pipes/validation.pipe';
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  Inject,
+  Param,
+  ParseIntPipe,
+  Post,
+  UseGuards,
+} from '@nestjs/common';
+import {
+  ApiBody,
+  ApiCreatedResponse,
+  ApiNoContentResponse,
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiTags,
+  ApiUnauthorizedResponse,
+} from '@nestjs/swagger';
+
+@ApiTags('organizations-safe')
+@Controller({
+  path: 'organizations/:organizationId/safes',
+  version: '1',
+})
+@UseGuards(AuthGuard)
+export class OrganizationSafesController {
+  public constructor(
+    @Inject(OrganizationSafesService)
+    private readonly organizationSafesService: OrganizationSafesService,
+  ) {}
+
+  @Post()
+  @ApiCreatedResponse({ description: 'Safes created successfully' })
+  @ApiBody({ type: CreateOrganizationSafeDto, isArray: true })
+  @ApiUnauthorizedResponse({
+    description: 'User unauthorize OR signer address not provided',
+  })
+  @ApiNotFoundResponse({ description: 'User not found.' })
+  public async create(
+    @Body(new ValidationPipe(CreateOrganizationSafesSchema))
+    body: Array<CreateOrganizationSafeDto>,
+    @Param(
+      'organizationId',
+      ParseIntPipe,
+      new ValidationPipe(RowSchema.shape.id),
+    )
+    organizationId: Organization['id'],
```

Comment:
We [need to explicitly type](https://github.com/safe-global/safe-client-gateway/pull/2361) `organizationId` (and other inferred `Param`s here otherwise the clients cannot generate helpers correctly.

I won't mark the other occruances here.